src/VecSim/algorithms/hnsw/hnswlib.h

```diff
@@ -449,6 +449,8 @@ void HierarchicalNSW<dist_t>::processCandidate_RangeSearch(tableint curNodeId,
     __builtin_prefetch(getDataByInternalId(*node_links));
     __builtin_prefetch(getDataByInternalId(*(node_links + 1)));
 
+    dist_t dynamic_range_search_boundaries = dyn_range * (1.0 + epsilon);
```

Comment:
Redundant. you can simply pass the calculated `dynamic_range_search_boundaries` here. you don't use the dynamic range in this function.